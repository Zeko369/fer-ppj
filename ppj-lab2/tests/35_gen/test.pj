a = a
a = 805*820 - a
