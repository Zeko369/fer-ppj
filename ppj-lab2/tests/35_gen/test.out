<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 a
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       IDN 1 a
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 a
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 805
       <T_lista>
        OP_PUTA 2 *
        <T>
         <P>
          BROJ 2 820
         <T_lista>
          $
      <E_lista>
       OP_MINUS 2 -
       <E>
        <T>
         <P>
          IDN 2 a
         <T_lista>
          $
        <E_lista>
         $
   <lista_naredbi>
    $
